uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/page.js

```diff
@@ -1,42 +1,38 @@
-var _ = require('lodash');
-var Promise = require('es6-promise').Promise;
+const _ = require('lodash');
+const Promise = require('es6-promise').Promise;
 
 module.exports = {
   /**
    * Initializes the page and removes any code warnings from the page
    *
-   * @param  {Object} chromy
+   * @param  {Object} puppet
    * @param  {Boolean} clearDialogs if true it will close modals and notifications
    * @return {Object}
    */
-  init: function (chromy, clearDialogs) {
+  init: async (puppet, clearDialogs) => {
     clearDialogs = typeof clearDialogs !== 'undefined' ? !!clearDialogs : true;
 
-    this.chromy = chromy;
-    !!this.waitForReady && this.waitForReady();
-
-    chromy.evaluate(function () {
-      return document.location.href;
-    })
-      .result(function (href) {
-        var isAdmin = href.indexOf('civicrm/') > 1;
-
-        if (isAdmin) {
-          chromy.evaluate(function () {
-            var errorsWrapper = document.querySelector('#content > #console');
-            errorsWrapper && (errorsWrapper.style.display = 'none');
-          });
-        } else {
-          chromy.evaluate(function () {
-            var errorsWrapper = document.querySelector('#messages .alert');
-            errorsWrapper && (errorsWrapper.style.display = 'none');
-          });
-        }
+    this.puppet = puppet;
+    !!this.waitForReady && await this.waitForReady();
+
+    let href = await this.puppet.evaluate(() => document.location.href);
```

Comment:
You don't need to use `this.puppet`. Also please move lets to the top of the function. Also, not sure these need to be `let`, maybe `const`?